dts/bindings/i3c/nxp,mcux-i3c.yaml

```diff
@@ -24,18 +24,25 @@ properties:
 
   clk-divider:
     type: int
-    description: Main clock divider for I3C
-    required: true
+    default: 0
```

Comment:
For S32Z this is not the hw default. I'd suggest to leave it here without a default value as it was, and use instead `DT_INST_PROP_OR(.., 0)`

Reply:
I've updated to do not add default value in dt